DataLayer/Context/GameBoardDbContext.Model.cs

```diff
@@ -9,10 +11,47 @@ protected override void OnModelCreating(ModelBuilder builder)
         {
             base.OnModelCreating(builder);
 
-            builder.Entity<IdentityUser>(
-                entity => entity.ToTable("User"));
+            builder.Entity<ApplicationUser>(
+                entity =>
+                {
+                    entity.Property(e => e.UserName).HasMaxLength(16);
 
-            builder.Entity<IdentityRole>( 
+                    entity.ToTable("User");
+                });
+
+            builder.Entity<Friendship>(
+                entity =>
+                {
+                    entity.HasKey(e => e.Id);
+                    entity.Property(e => e.Id)
+                        .HasMaxLength(32)
+                        .IsRequired()
```

Comment:
`HasMaxLength` on an `int` makes no sense. Also, a PK constraint already implies `IsRequired`, so it's redundant.

Reply:
Fixed